src/jetton.h

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: 2025 Ledger SAS
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#pragma once
+
+#ifdef HAVE_HARDCODED_JETTONS
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <inttypes.h>
+
+#define JETTON_NAME_MAX_SIZE 8UL
```

Comment:
Might be worth it to indicate where this value comes from, with a spec link for example

Reply:
Done